src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -249,10 +266,12 @@ def csv(self, path: str) -> DataFrame:
 
         Args:
             path: The stage location of a CSV file, or a stage location that has CSV files.
```

Comment:
```suggestion
            path: A path pointing to a local CSV file, a local directory that has CSV files, a CSV file on a stage, or a stage location that has CSV files.
```
Same below, but others don't support multiple files.